docs/scraping/providers/powerbi-embedded.md

```diff
@@ -0,0 +1,34 @@
+---
+tags:
+  - Scraper
+  - Resource Discovery
+  - PowerBi
+---
+
+# Azure PowerBi Embedded
+
+![Availability Badge](https://img.shields.io/badge/Available%20Starting-v2.9-green.svg)![Resource Discovery Support Badge](https://img.shields.io/badge/Support%20for%20Resource%20Discovery-Yes-green.svg)
+
+You can declare to scrape a PowerBi Embedded capacity via the `PowerBiEmbedded` resource-type.
```

Comment:
Is `PowerBiEmbedded` the correct fit? It looks like it's only for dedicated power BI?

Also, is Power BI Embedded not deprecated? I might be remembering this wrong or confuse with another Azure offering of Power BI

Reply:
@tomkerkhove any preference on the naming?

Reply:
sounds good, I updated the naming here, I'll make an update to the main PR later tonight